docs/FAQ.md

```diff
@@ -318,3 +318,166 @@ kubectl delete ns calico-apiserver calico-system
 2. Set `enabled: false` for `id: aws-load-balancer-controller` in the **layer2-k8s/helm-releases.yaml** file
 3. Set `enabled: true` for `id: external-dns`, `id: cert-manager`, `id: cert-mananger-certificate`, `id:cert-manager-cluster-issuer` in the **layer2-k8s/helm-releases.yaml** file
 4. Run `terraform apply` in the layer2-k8s folder
+
+## Apply using terraform
+
+### S3 state backend
+
+By default, you can use local state for this project, but we suggest you to use S3 backend.
+
+<details>
+  <summary>S3 bucket for remote state</summary>
+
+  Set `STATE_BUCKET_NAME` and `STATE_BUCKET_REGION`, then create S3 bucket:
+
+  ```bash
+  $ aws s3api create-bucket \
```

Comment:
It's better to enable versioning on bucket level because it's disabled by default

Reply:
done)